cmd/kola/kola.go

```diff
@@ -250,12 +269,50 @@ func (i item) String() string {
 			}
 		}
 		i.Platforms = platforms
+	} else {
+		i.ExcludePlatforms = []string{""}
 	}
 	if len(i.Platforms) == 0 {
-		i.Platforms = []string{"all"}
+		if listJSON {
+			i.Platforms = kolaPlatforms
+		} else {
+			i.Platforms = []string{"all"}
+		}
 	}
 	if len(i.Architectures) == 0 {
-		i.Architectures = []string{"all"}
+		if listJSON {
+			i.Architectures = kolaArchitectures
+		} else {
+			i.Architectures = []string{"all"}
+		}
```

Comment:
I don't know this codebase well, so this is a question - is the idea here that when emitting JSON, we want to declare what architectures we support, but when evaluating the tests, we want to assume that the test supports all of them?

Offhand, it feels a bit weird to do it this way versus having e.g. toplevel JSON elements `platforms: $kolaPlatforms, architectures: $kolaArchitectures` and then the tests default to "all" the same as they do internally?

Reply:
With CL dropping arm64 support (and mantle subsequently following) there is only one architecture.

As far as why it's `all` on the stdout approach vs naming them all in the JSON, I found that there'd be more value in enumerating them for CI in JSON while still maintaining the current functionality of using `all`.

Reply:
> With CL dropping arm64 support (and mantle subsequently following) there is only one architecture.

Side note: mantle isn't going to drop arm64 support because we'll need it in the future.